audio/midi_drivers/FluidSynthMidiDriver.cpp

```diff
@@ -108,11 +121,11 @@ int FluidSynthMidiDriver::open() {
 }
 
 void FluidSynthMidiDriver::close() {
-	while (!_soundFont.empty()) {
-		int soundfont = _soundFont.top();
+	//-- Do not sfunload the SoundFonts, this causes messages :
+	//--   fluidsynth: warning: No preset found on channel # [bank=# prog=#]
+	//-- And delete_fluid_synth unloads the SoundFonts anyway.
+	while (!_soundFont.empty())
 		_soundFont.pop();
```

Comment:
Again, change loop to `_sounfFont.clear();`

Reply:
Sorry, answered above. I cannot, _soundFont is not a std::vector, it is a std::stack, it has no .clear(), I tried and it failed.

Reply:
Oh, right. Hm, nevermind then.